lifecycle-operator/main.go

```diff
@@ -347,6 +358,9 @@ func main() {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
+
+	// Set the metric value as soon as the operator starts
+	keptnLifecycleActive.Add(context.Background(), 1)
```

Comment:
I would add this after the line `setupLog.Info("Keptn lifecycle-operator is alive")`